src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -314,23 +315,44 @@ def refreshCache(self):
                 possibleLocations = set()
 
                 # all files in job have same location (in se names)
-                rawLocations = loadedJob["input_files"][0]["locations"]
+                #rawLocations = loadedJob["input_files"][0]["locations"]
+                #rawLocations = loadedJob.get("inputDatasetLocations") or []
+                #possibleLocations.update(rawLocations)
+                from pprint import pformat
+                rawLocations = loadedJob.get("inputDatasetLocations") or []
```

Comment:
but we know that the PNN now comes back with FWJR so locations should be picking this up but is not. I guess this will be solved with the resolution of #6004